scripts/backup

```diff
@@ -6,13 +6,17 @@
 # IMPORT GENERIC HELPERS
 #=================================================
 
+# Keep this path for calling _common.sh inside the execution's context of backup and restore scripts
 source ../settings/scripts/_common.sh
 source /usr/share/yunohost/helpers
 
 #=================================================
 # MANAGE SCRIPT FAILURE
 #=================================================
 
+ynh_clean_setup () {
```

Comment:
Not needed as stated by @tituspijean here : https://github.com/YunoHost-Apps/dokuwiki_ynh/pull/92#issuecomment-1223069994

( I had no idea what this code does before asking )

may be for to add some explanation in "example_ynh" about it? Or just remove it
https://github.com/YunoHost/example_ynh/blob/master/scripts/install#L16-L18